src/components/DeleteClientModal/DeleteClientModal.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import CheckIcon from '@mui/icons-material/Check';
+
```

Comment:
Be sure to add JSDocs here underneath the imports. You can check out the other components to see how that's done.

Reply:
Yeah, we should add some JSDoc to document the new component. Something basic should do, check out the JSDoc in `AddClientModal.jsx` between Line 13 to 18.